### PR TITLE
Redo README and add current status note

### PR DESCRIPTION
**Pull request by willkg**
We no longer support Socorro and we're making big changes. Seems prudent to let
users know the status and what the last good commit was.

### CODE REVIEW

**Comment by willkg** [comment]
@lonnen Is the text here clear enough? My concern is that I'm going to go delete all the collector code and given that users continue to use Socorro and ask for support, I don't think we've been super clear about how we're not supporting Socorro anymore and how we're making backwards-incompatible changes.

So this text makes that clearer and gives them a sha for the last good commit (for some definition of "last good commit").

File: README.rst [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+<http://code.google.com/p/google-breakpad/>`_.
+
+
+Support
+=======
+
+This is a very Mozilla-specific product. We do not currently have the capacity
+to support external users. If you are looking to use Socorro for your product,
+maybe you want to consider alternatives like `electron/mini-breakpad-server
+<https://github.com/electron/mini-breakpad-server>`_.
```
Comment (lonnen) [inline README.rst]: let's list https://github.com/wk8/sentry_breakpad as an alternative also

File: README.rst [hunk 2]
```diff
@@ -0,0 +1,73 @@
+=======
+Socorro
+=======
+
+Socorro is a set of services for collecting, processing, and displaying crash
```
Comment (lonnen) [inline README.rst]: consider adding a word or two to emphasize Mozilla-specific

File: README.rst [hunk 3]
```diff
@@ -0,0 +1,73 @@
...
+Devs hang out in the Socorro/Breakpad IRC channel:
+`<irc://irc.mozilla.org/breakpad>`_
+
+
+Development
+===========
+
+Current deployment status: https://whatsdeployed.io/s-7M7
+
+Infrastructure (AWS) code: https://github.com/mozilla/socorro-infra
```
Comment (lonnen) [inline README.rst]: should we note that this repo is similarly deprecated?

**Comment by lonnen** [comment]
Rather than do a date-base series of updates as we pull components, consider anointing a tag as the last holistic release of the system.

**Comment by willkg** [comment]
I'll switch it to a tag--that's a good idea.

I was thinking "this is the last known good one" after which we have carte blanche to make any changes we want. I wasn't planning on doing further status updates.

File: README.rst (hunk 3 above)
Comment (willkg) [inline README.rst]: I have no idea if it is or not. I thought it was Mozilla-centric and there was no expectation that it was supported in any way.

Comment (willkg) [inline README.rst]: I'll add a note about how this is Mozilla-centric and not supported, too.

Comment (willkg) [inline README.rst]: https://github.com/mozilla/socorro-infra/pull/306

**Comment by willkg** [comment]
@lonnen ^^^ Does that look good?

**Comment by willkg** [comment]
I think I addressed all of @lonnen's comments and that this is fine. If not, we can do another PR.

**Comment by lonnen** [comment]
Belated r+

**Pull request merged**